en/docs/adv_examples/bert.md

```diff
@@ -1,48 +1,48 @@
 
-## 模型概述
-BERT(Bidirectional Encoder Representations from Transformers)是NLP领域的一种新型预训练模型。本案例中，基于论文[BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding](https://arxiv.org/abs/1810.04805)实现了BERT模型的OneFlow版本。本案例中，基于论文[BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding](https://arxiv.org/abs/1810.04805)实现了BERT模型的OneFlow版本。
+## Summary
+BERT(Bidirectional Encoder Representations from Transformers) is a technique for NLP. In our case, we implements BERT based on the paper [BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding](https://arxiv.org/abs/1810.04805) using OneFlow.
 
-### 模型架构
+### Model
 | **Model** | **Hidden layers** | **Hidden unit size** | **Attention heads** | **Feedforward filter size** | **Max sequence length** | **Parameters** |
 |:---------:|:-----------------:|:--------------------:|:-------------------:|:---------------------------:|:-----------------------:|:--------------:|
 | BERTBASE  |    12 encoder     |         768          |         12          |          4 x  768           |           512           |      110M      |
 
-BERT在实际应用中往往分为两步：
+There are commonly two steps in BERT:
 
-* 首先，预训练得到BERT语言模型；
+* First, BERT pre-trained is obtained by pre-training;
 
-* 然后，为满足下游应用，在得到的BERT语言模型的基础上，多加一层网络，并进行微调，得到下游应用。
+* Then, on the basis of the obtained pre-trained model, an additional layer of network is added and finetuned to get the downstream application.
 
 
-## 快速开始
-### 获取相关数据集
-我们提供了完成BERT预训练及SQuAD微调的[OFRecord数据集及相关数据文件](https://oneflow-static.oss-cn-beijing.aliyuncs.com/oneflow-tutorial-attachments/bert_squad_dataset.zip)，可以通过以下命令下载并解压：
+## Quickstart
+### Get dataset
+We provide [OFRecord data set and relevant other files](https://oneflow-static.oss-cn-beijing.aliyuncs.com/oneflow-tutorial-attachments/bert_squad_dataset.zip), you can get and unzip it by running commands below:
 
 ```bash
 wget https://oneflow-static.oss-cn-beijing.aliyuncs.com/oneflow-tutorial-attachments/bert_squad_dataset.zip
 unzip bert_squad_dataset.zip
 ```
-解压后的文件目录清单如下：
+The list of files is as follows:
 
-* bert_config.json、vocab.txt：制作prediction json文件需要的文件，来自[google bert](https://github.com/google-research/bert)
+* bert_config.json、vocab.txt：Files needed to generate "prediction json" file from [google bert](https://github.com/google-research/bert)
 
-* dev-v1.1/、dev-v1.1.json：SQuAD检验集，用于打分
+* dev-v1.1/, dev-v1.1.json：SQuAD test set for evaluation
 
-* part-0：预训练集样本（40个样本）
+* part-0：pre-trained training set contains 40 samples
 
-* train-v1.1：SQuAD训练集，已经转为ofrecord数据集格式
+* train-v1.1：SQuAD training set that has been coverted to OFRecords
 
-以上各个文件将在下文的预训练任务、SQuAD微调中使用到。
+The above files will be used in the following pre-training tasks and squad finetune.
```

Comment:
The above files will be used in the following pretraining tasks and squad finetune.